Makefile.am

```diff
@@ -12,7 +12,7 @@ AM_LDFLAGS      = $(EXTRA_LDFLAGS) $(CODE_COVERAGE_LIBS)
 # Initialize empty variables to be extended throughout
 lib_LTLIBRARIES =
 noinst_LTLIBRARIES =
-EXTRA_DIST =
+EXTRA_DIST = test/fuzz/main-sapi.cpp
```

Comment:
This should probably go to Makefile-fuzz.am to keep the test and fuzz separate from the rest of the code. As the comment above says this only
initializes empty variables...

Reply:
I think it will be simpler, if there is eventually one EXTRA_DIST on a single place including all relevant files, rather than splitting it over different places.

Reply:
I'm not sure about that. Currently the appropriate files are appended to EXTRA_DIST "by component", which is nice and logical.
In some cases this might be conditional. I would prefer this to be in Makefile-fuzz.am.

Reply:
I agree. I also prefer the `+=` convention where all settings of a component are bundled together.